test/test_transforms_tensor.py

```diff
@@ -608,6 +510,86 @@ def test_to_grayscale(device, Klass, meth_kwargs):
     )
 
 
+@cpu_only
+@pytest.mark.xfail()
+@pytest.mark.parametrize(
+    'in_dtype,out_dtype', [
+        (int_dtypes() + float_dtypes(), int_dtypes() + float_dtypes())
+    ]
+)
```

Comment:
there are 2 nested for loops here so we should not parametrize over tuples, instead we should have 2 separate paraemtrization to have a cross-produce:

```py
@pytest.mark.parametrize('in_dtype', int_dtypes() + float_dtypes())
@pytest.mark.parametrize('out_dtype', int_dtypes() + float_dtypes())
```